Cargo.toml

```diff
@@ -30,13 +30,14 @@ lz4 = { version = "1.23", optional = true }
 parking_lot = "0.12"
 rand = "0.8"
 rustls = { version = "0.23", optional = true, default-features = false, features = ["logging", "ring", "std", "tls12"] }
+serde = "^1.0.210"
```

Comment:
This doesn't seem to be used anywhere?

Reply:
There's a version conflict introduced by strict bounds on `serde` in `rsasl` and `criterion` (I think), this is needed for version resolution.

Reply:
I'll have a look at that :eyes:

Reply:
With this line removed, I have zero issues. Could you try to remove your `Cargo.lock` and see if it works w/o this line? If not, what exact issue are you facing (i.e. can you post the error message)?

Reply:
Huh. You're right. My bad. I did get some weird error but removing Cargo.lock worked.